content/deploy/choose_cloud.md

```diff
@@ -0,0 +1,70 @@
+# Choosing where your JupyterHub will run
+
+## Should I deploy on my hardware or in the cloud?
+
+The first thing you'll decide is *where* your JupyterHub will run. The first
+question to ask is "will I run my JupyterHub on my own hardware, or will I
+run it in the cloud?" Running a JupyterHub on your own hardware is your best
+option if you have some technical support for doing so. If you're working with
+limited technical resources, running a JupyterHub in the cloud is usually simpler.
+The rest of this guide assumes you'd like to run a JupyterHub in the cloud.
+
+(however, note that you can still [demo the course environment locally](deploy_locally.html))
+
+## Choosing a cloud provider and service
+
+Cloud providers (e.g. Microsoft Azure, Google Cloud, or Digital Ocean) are where you'll
+run your JupyterHub. These companies basically own really large computers, and what
+you do is purchase some resources on those computers to run whatever you'd like (in this
+case, a JupyterHub). However, there are often a few different kinds of offerings that
+cloud providers provide, with various pros and cons.
+
+> **An aside on using free hosted services vs. creating your own deployments**. Many cloud
+  providers offer a "free" and customized notebook service for basic usage. While these are
+  appealing on their surface, they provide a number of challenges down the road, such as
+  getting your students used to the vendor-specific interface rather than an open-source version.
+  The Jupyter Project (and the Data 8 approach) aims to maximize your choice and flexibility in
+  how and where you deploy Jupyter environments on shared infrastructure, and for this reason
+  we do not recommend using most fully-managed "free" services.
```

Comment:
I would suggest framing this similar to the others with "pros" and "cons". For under-resourced instructors, using a managed "free" service may be the most feasible approach for them in the first deployment of a course. We don't need to spend much time discussing how to do this, and focus in on DIY approaches, but I think there are quite a number of instructors for whom launching their own hub is a complete non-starter. If they can get started on a service, and then transition, perhaps in the next delivery of the course to using their own deployment, then that is still a net-win.